### ISSUE TITLE
Calling `res.blob()` or `res.arrayBuffer()` never ends.

### ISSUE THREAD

**Issue by iamandrewluca** [gustavoguichard/make-service#46]
Calling `await response.blob()` or `await response.arrayBuffer()` never ends.
In the example below "After blob" message never shows.
Please see video for more details

```js
const response = await service.get(url);

console.log('Before blob');
const blob = await response.blob();
console.log('After blob');
```

https://www.youtube.com/watch?v=FX5uUwhs_8M

**Comment by gustavoguichard** [comment]
Got my attention, @iamandrewluca

**Comment by gustavoguichard** [comment]
@iamandrewluca thank you for such a good issue!
I'm gonna need some time to investigate on it. We are using Proxy to "monkey-patch" the Response object and it might be the source of this problem.
I'll try to find a solution ASAP.

Thanks for being such a good user/contributor

**Comment by iamandrewluca** [comment]
I also looked through the codebase but did not see the monkey patch. I will check again.

**Comment by iamandrewluca** [comment]
Oh boy 😄 I tried all the possible variations )) None Works

I'm reading in this blog post, that some native objects cannot be proxied as usual.

https://javascript.info/proxy

**Comment by iamandrewluca** [comment]
I will do it tomorrow!

**Comment by gustavoguichard** [comment]
@iamandrewluca I just released `make-service@3.0.0` with the solutions to your 2 latest issues ;)

**Comment by iamandrewluca** [comment]
![let's go](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExcGZ3dXg2N3RjMzU5NGN2b2Y0NmdueW54N3V2Ymg0eHB1aTJjYTFubSZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/7WvAUvZZTRpSuudobh/giphy.gif)
